[2024-01-31 15:30:45,439] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_postgresql.aggregate_dim_table manual__2024-01-31T15:30:19.767407+00:00 [queued]>
[2024-01-31 15:30:45,445] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_postgresql.aggregate_dim_table manual__2024-01-31T15:30:19.767407+00:00 [queued]>
[2024-01-31 15:30:45,445] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-01-31 15:30:45,445] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-01-31 15:30:45,446] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-01-31 15:30:45,454] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): aggregate_dim_table> on 2024-01-31 15:30:19.767407+00:00
[2024-01-31 15:30:45,458] {standard_task_runner.py:52} INFO - Started process 10153 to run task
[2024-01-31 15:30:45,460] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_postgresql', 'aggregate_dim_table', 'manual__2024-01-31T15:30:19.767407+00:00', '--job-id', '196', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpnbqny4a5', '--error-file', '/tmp/tmps9ytsaj9']
[2024-01-31 15:30:45,461] {standard_task_runner.py:80} INFO - Job 196: Subtask aggregate_dim_table
[2024-01-31 15:30:45,497] {task_command.py:369} INFO - Running <TaskInstance: etl_postgresql.aggregate_dim_table manual__2024-01-31T15:30:19.767407+00:00 [running]> on host d99f9e65d396
[2024-01-31 15:30:45,543] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_postgresql
AIRFLOW_CTX_TASK_ID=aggregate_dim_table
AIRFLOW_CTX_EXECUTION_DATE=2024-01-31T15:30:19.767407+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-31T15:30:19.767407+00:00
[2024-01-31 15:30:48,346] {logging_mixin.py:115} INFO - Successfully install sqlalchemy
[2024-01-31 15:30:51,165] {logging_mixin.py:115} INFO - Successfully install sqlalchemy
[2024-01-31 15:30:53,988] {logging_mixin.py:115} INFO - Successfully install pymysql
[2024-01-31 15:30:57,035] {logging_mixin.py:115} INFO - Successfully install sqlalchemy
[2024-01-31 15:30:57,084] {logging_mixin.py:115} INFO - <sqlalchemy.engine.cursor.CursorResult object at 0xffff90056250>
[2024-01-31 15:30:57,092] {python.py:173} INFO - Done. Returned value was: insert dim table
[2024-01-31 15:30:57,112] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_postgresql, task_id=aggregate_dim_table, execution_date=20240131T153019, start_date=20240131T153045, end_date=20240131T153057
[2024-01-31 15:30:57,134] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-01-31 15:30:57,156] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
